src/main/kotlin/no/nav/syfo/consumer/PdlConsumer.kt

```diff
@@ -40,29 +40,33 @@ class PdlConsumer(env: Environment, stsConsumer: StsConsumer) {
     }
 
     suspend fun getFnr(aktorId: String) : String? {
-        val stsToken = stsConsumer.getToken()
-        val bearerTokenString = "Bearer ${stsToken.access_token}"
-        val graphQuery = this::class.java.getResource("/pdl/hentIdenter.graphql").readText().replace("[\n\r]", "")
-        val requestBody = PdlRequest(graphQuery, Variables(aktorId))
+        val response = callPdl(IDENTER_QUERY, aktorId)
 
-        val response = try {
-            client.post<HttpResponse>(pdlBasepath) {
-                headers {
-                    append(TEMA_HEADER, OPPFOLGING_TEMA_HEADERVERDI)
-                    append(HttpHeaders.ContentType, APPLICATION_JSON)
-                    append(HttpHeaders.Authorization, bearerTokenString)
-                    append(NAV_CONSUMER_TOKEN_HEADER, bearerTokenString)
-                }
-                body = requestBody
+        return when (response?.status) {
+            HttpStatusCode.OK -> {
+                response.receive<PdlIdentResponse>().data?.hentIdenter?.identer?.first()?.ident
+            }
+            HttpStatusCode.NoContent -> {
+                log.error("Could not get oppfolgingstilfelle: No content found in the response body")
+                null
+            }
+            HttpStatusCode.Unauthorized -> {
+                log.error("Could not get oppfolgingstilfelle: Unable to authorize")
+                null
+            }
+            else -> {
+                log.error("Could not get oppfolgingstilfelle: $response")
```

Comment:
"Could not get oppfolgingstilfelle": er det riktig melding?

Reply:
Fikset 👍